Ch02/Ch02.md

```diff
@@ -1176,241 +1176,241 @@ using (var session = store.OpenSession())
+> RavenDB, we're storing document Ids separately from the actual document data, and we're making sure to coalesce the pages
 > holding the document keys so we have a good locality of reference. 
 > 
-> Even with a database that holds a hundred million documents, the whole of the document ids data is likely to be memory 
-> resident, which turns the cost of finding a particular document to be extremely cheap. 
+> Even with a database that holds a hundred million documents, the whole of the document Id data is likely to be memory 
+> resident, which makes the cost of finding a particular document extremely cheap. 
 
-The one option that you need to be cautious of is the use of the identity generation method. You need to be careful not to use
-it without careful consideration and analysis. Identity requires network roundtrips to generate the next value, and it also 
+The one option you need to be cautious of is the identity generation method. Be careful not to use
+it without careful consideration and analysis. Identity requires network roundtrips to generate the next value, and it will 
 become unavailable if the node cannot communicate with the other members in the cluster. 
 
-## Documents metadata
+## Document metadata
 
-The document data is composed of whatever it is that you're storing in the document. For the order document, that would be
-the shipping details, the order lines, who the customer is, the order priority, etc. But you also need a place to store 
-additional information, not related to the document itself, but _about_ the document. This is where the metadata comes into
-place.
+Document data is composed of whatever it is that you're storing in the document. For the order document, that would be
+the shipping details, the order lines, who the customer is, the order priority, etc. You also need a place to store 
+additional information that's unrelated to the document itself but is rather _about_ the document. This is where metadata comes into
+play.
 
-The metadata is also a JSON format, just like the document data itself. RavenDB reserves metadata property names that starts
-with `@` for its own use, but you are free to use anything else. By convention, the users' custom metdata properties are 
-using Pascal-Cased naming convention. In other words, we separate words with a dash and the first letter of each word is
-capitalized, everything else is in lower case. RavenDB's internal metadata properties are using `@` prefix, all lower cased
-and separating words using a dash (e.g: `@last-modified`).
+The metadata is also in JSON format, just like the document data itself. RavenDB reserves for its own use metadata property names that 
+start with `@` , but you're free to use anything else. By convention, users' custom metadata properties 
+use Pascal case capitalization. In other words, we separate words with a dash, and the first letter of each word is
```

Comment:
We actually meant: "Capitalized words separated by hypens here" I agree that PascalCase sin't the right term, and kebab-case is funny, but it isn't accurate with regards to the casing.